src/constants.cc

```diff
@@ -160,5 +170,27 @@ bool ClientPinResponseContains(int64_t key) {
   return key >= 0x01 && key <= 0x05;
 }
 
+cbor::Value CborValue(CredentialManagementResponse response) {
+  return cbor::Value(static_cast<uint8_t>(response));
+}
+
+bool CredentialManagementResponseContains(int64_t key) {
+  return key >= 0x01 && key <= 0x0B;
+}
+
+cbor::Value CborValue(LargeBlobsResponse response) {
+  return cbor::Value(static_cast<uint8_t>(response));
+}
+
+bool LargeBlobsResponseContains(int64_t key) { return key == 0x01; }
+
+cbor::Value CborValue(BioEnrollmentResponse response) {
+  return cbor::Value(static_cast<uint8_t>(response));
+}
+
+bool BioEnrollmentResponseContains(int64_t key) {
+  return key >= 0x01 && key <= 0x08;
```

Comment:
I guess doing it like that is ok because enums are probably monotonic over time (we only add new values) and we would see that we can't use the new ones and update this function. An alternative if we want to avoid this last update would be to add `kBioEnrollmentReserved` as the last value (and similarly for other enums) and test `0 < key && key < kBioEnrollmentReserved`. As you want.

Reply:
Wouldn't that possibly trigger compiler warning when I `switch` over the enum and ignore `kBioEnrollmentReserved`?
This implementation is really just an implementation detail, as holes in the range are definitely possible in the future. Therefore, I'd prefer not to have `Reserved` variants.

Reply:
Indeed, we would need to add CHECK(false); on the `Reserved` variants which is not great. And if holes are expected it makes it more complicated. It's too bad there's no C++ helpers for that (like there is for proto enums or the `num_enum` Rust crate). Let's keep it like that then and have to duplicate the information in the enum definition and this checking function.